src/main/java/org/folio/rest/impl/RestrictionsHelper.java

```diff
@@ -0,0 +1,23 @@
+package org.folio.rest.impl;
+
+import static org.folio.invoices.utils.ErrorCodes.APPROVED_OR_PAID_INVOICE_DELETE_FORBIDDEN;
+import static org.folio.rest.acq.model.Invoice.Status.APPROVED;
+import static org.folio.rest.acq.model.Invoice.Status.PAID;
+
+import org.folio.HttpStatus;
+import org.folio.invoices.rest.exceptions.HttpException;
+import org.folio.rest.jaxrs.model.Invoice;
+
+import java.util.concurrent.CompletableFuture;
+
+public class RestrictionsHelper {
+  private RestrictionsHelper() {
+  }
```

Comment:
Please move it in **org.folio.invoices.utils** package and rename InvoiceRestrictionsUtil

Reply:
done